splunk_add_on_ucc_framework/global_config.py

```diff
@@ -91,6 +91,8 @@ def from_app_manifest(
                 "restRoot": app_manifest.get_addon_name(),
                 "displayName": app_manifest.get_title(),
                 "version": app_manifest.get_addon_version(),
+                # adding this by default as to omit reading of app.conf
+                "checkForUpdates": True,
```

Comment:
It is defaulted to `True` in `schema.json`, why do we need to explicitly add it here?

Reply:
Yes, by default `app.conf` would have this property, but since we are generating a new globalConfig I thought we should expose this as well. We can remove it if not required.

Reply:
Updated the code.